src/features/user/TreeMapper/components/PlantLocationPage.tsx

```diff
@@ -92,6 +107,25 @@ export default function PlantLocationPage({
       </div>
 
       <LocationDetails {...DetailProps} />
+      <br />
+      <br />
+      {species && (
+        <>
+          <b>Sample Tree</b>
+          <div className={styles.sampleTreesContainer}>
+            {species.map((tree, index) => (
+              <div className={styles.singleTree} key={tree.id}>
+                <p>
+                  {index + 1} .{' '}
```

Comment:
Why not use an [ordered list](https://www.w3schools.com/html/html_lists.asp) with the `<ol>` tag?